custom_typings/dom.d.ts

```diff
@@ -0,0 +1,11 @@
+declare interface Window {
+  customElements: CustomElementRegistry;
```

Comment:
I see that that tool pulls DOM definitions from Edge. Does Edge pick up IDL for features it hasn't implemented yet, like Shadow DOM and Custom Elements?